.circleci/config.yml

```diff
@@ -1226,6 +1226,36 @@ jobs:
             python3 ./tools/release/publish_release.py --no-build-deb --no-build-homebrew --github-token ${GITHUB_TOKEN} --chocolatey-token ${CHOCO_TOKEN} --use-existing-release --version ${VERSION} --output-dir artifacts
           shell: bash.exe
 
+  publish_docs:
+    environment:
+      <<: *linux_environment
+    working_directory: "/home/circleci/buck"
```

Comment:
Why do we need to set the working directory? Aren't we just working from the root of the clone?

Reply:
This instructs CircleCI where to clone the code to.